gfeloqua.class.php

```diff
@@ -98,8 +98,6 @@ public function init(){
 		// entry detail
 		add_action( 'gform_entry_detail', array( $this, 'entry_notes' ), 10, 2 );
 
-		// entry meta (for column)
-		add_action( 'gform_entry_meta', array( $this, 'add_success_meta' ), 10, 2 );
```

Comment:
The GF add-on framework already includes the gform_entry_meta filter hooked to get_entry_meta() so there is no need to add the hook again. Using the frameworks built-in method for adding the entry meta helps ensure add-ons implement common features in a standard way making it easier for other developers to create their own integrations not just with Gravity Forms but with any add-on which extends Gravity Forms.